src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -79,13 +79,13 @@ public async Task DeployCdkProject(OrchestratorSession session, CloudApplication
 
             _interactiveService.LogMessageLine("Deploying AWS CDK project");
 
+            var appSettingsFilePath = Path.Combine(cdkProjectPath, "appsettings.json");
+
             // Ensure region is bootstrapped
-            await _commandLineWrapper.Run($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion}",
+            await _commandLineWrapper.Run($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion} -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\"",
```

Comment:
Looks like with cdk v2 it actually runs the cdk project in the directory because of there being a cdk.json file. Since we were not passing the appsetings.json file the .NET project would fail to run causing bootstrapping to fail.